lib/python-beta/src/bailo/helper/model.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+import os
+from typing import Any
+
+from bailo.core import Client, ModelVisibility
+from bailo.helper.utils import retrieve_model
+
+
+class Model:
+    def __init__(
+        self,
+        client: Client,
+        name: str,
+        description: str,
+        visibility: ModelVisibility | None = ModelVisibility.Public,
+        model_card: dict[str, Any] | None = None
+    ):
+        self.client = client
+        self.name, self.description, self.visibility = name, description, visibility
+        self.model_id = None
+        self.model_version = None
+        self.model_card_version = None
+        self.model_card_schema = 'minimal-general-v10-beta'
+        self.__model_card_original = model_card
+        self.__model_card = model_card
+        self.__local_dir = f"./temp_{name.strip(' ')}"  # property tbc
+        self.__mc_change = 0
+        self.__file_change = 0  # property tbc
+
+    @classmethod
```

Comment:
Use a static method instead of a class method here, and create the class by name 'Model'.  It's more obvious than using the generic `cls` throughout the code.

Reply:
In the `release` PR it was mentioned this wouldn't be inherited by classes that extend this.  So maybe keeping it as a `classmethod` is better.  Could we at least give the variable name a better name than `cls` though? :)